frontend/src/metabase/css/query_builder.module.css

```diff
@@ -125,7 +125,11 @@
 }
 
 .Loading {
-  background-color: color-mod(var(--mb-color-bg-white) alpha(-28%));
+  background-color: color-mix(
+    in srgb,
+    var(--mb-color-bg-white) 72%,
```

Comment:
should we do 70% instead for a nicer number?

Reply:
Good idea!